boa3/internal/analyser/constructanalyser.py

```diff
@@ -44,16 +44,13 @@ def visit_Call(self, call: ast.Call) -> ast.AST:
                     break
 
             if to_script_hash is not None:
-                from boa3.internal.constants import SYS_VERSION_INFO
                 from boa3.internal.model.type.type import Type
                 types = {
                     Type.int.identifier: int,
                     Type.str.identifier: str,
                     Type.bytes.identifier: bytes
                 }
-                literal: tuple = ((ast.Constant,)
-                                  if SYS_VERSION_INFO >= (3, 8)
-                                  else (ast.Num, ast.Str, ast.Bytes))
+                literal: tuple = (ast.Constant,)
```

Comment:
this variable is only used in one place (line 58), now that the value doesn't depend on the `if else` result it's not necessary anymore